CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.15)
+include_guard(GLOBAL)
+
+project(batotp)
+
+option(BUILD_EXAMPLES ON)
```

Comment:
When I followed the steps in the readme, the tests were not compiled. I had to change this line to 
`set(BUILD_EXAMPLES ON)`